docs/_sass/_search-bar.scss

```diff
@@ -179,8 +179,9 @@ label.search-label {
     display: none;
 }
 
-.gs-title {
+.gs-title * {
     font-weight: bold;
+    color: var(--color-link) !important;
```

Comment:
We need to use `!important` to override the colors set in google search console. They are static and we can't have them dynamic. So this is the only way

Reply:
Is `*` really required?

Reply:
yes because there are like 5 different classes / pseudo classes under title that need this style applied.

Reply:
Makes sense.
Could you please add QA steps

Reply:
@rushatgabhane ^

Reply:
added!

Reply:
thanks